docs/latest.md

```diff
@@ -370,7 +370,7 @@ pkg_tar(<a href="#pkg_tar-name">name</a>, <a href="#pkg_tar-deps">deps</a>, <a h
 | <a id="pkg_tar-create_parents"></a>create_parents |  -   | Boolean | optional |  `True`  |
 | <a id="pkg_tar-empty_dirs"></a>empty_dirs |  -   | List of strings | optional |  `[]`  |
 | <a id="pkg_tar-empty_files"></a>empty_files |  -   | List of strings | optional |  `[]`  |
-| <a id="pkg_tar-extension"></a>extension |  -   | String | optional |  `"tar"`  |
+| <a id="pkg_tar-extension"></a>extension | The extension of the generated file. If `"gz"`, `"bz2"`, or `"xz"`, the tarball will also be compressed using that tool, and is mutually exclusive with `compressor`. Note that `xz` may not be supported based on the Python toolchain. | String | optional |  `"tar"`  |
```

Comment:
> Note that `xz` may not be supported based on the Python toolchain.

I'm not 100% sure this is true. The source has a TODO to confirm `xz` availabilty, but it's been there for > 3 years now and `HAS_XZ_SUPPORT` is True.

https://github.com/bazelbuild/rules_pkg/blob/8c7c2cf7f1742a1fb199d0164b800d2939fa9fef/pkg/private/tar/tar.bzl#L29-L31

Maybe I just shouldn't mention that `xz` might not be supported?

Reply:
This is fine.

Reply:
But the problem is that it won't work if you change it here. 
It will get lost on the next release update when the file is generated. You need to change it in the rule souce.

Reply:
Ah, thanks for letting me know. I've updated the starlark source instead, please LMK if there's something else I'm supposed to do.